Makefile

```
@@ -18,9 +18,15 @@ worker: files/install-deps-worker.yaml files/recipe-worker.yaml
 # Make sure you have latest docker.io/usercont/packit:prod prior to running this
 worker-prod: files/install-deps-worker.yaml files/recipe-worker.yaml
 	$(CONTAINER_ENGINE) build --rm -t $(WORKER_IMAGE_PROD) -f Dockerfile.worker.prod .
+
 worker-prod-push: worker-prod
 	$(CONTAINER_ENGINE) push $(WORKER_IMAGE_PROD)
 
+# in case of need change ip this is default so whould work, also be sure docker is logged to local oc registry 'docker login 172.30.1.1:5000'
+worker-local-push: worker
+	$(CONTAINER_ENGINE) tag $(WORKER_IMAGE) 172.30.1.1:5000/packit-stg/packit-worker:dev && \
```

```suggestion
	$(CONTAINER_ENGINE) tag $(WORKER_IMAGE) 172.30.1.1:5000/packit-stg/packit-worker:dev
```
no need to, make will take care of the serialization